### PR TITLE
Cleanup READMEs and Man pages

### PR DESCRIPTION
**Pull request by rhatdan**
Alphabatize correctly and add missing content.  Also add video.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
@mheon @baude PTAL

**Comment by mheon** [comment]
LGTM. Just docs changes, let's get this merged quick.

**Comment by rhatdan** [comment]
Yes as soon as travis passes, no reason for E2E.

File: docs/kpod.1.md [hunk 1]
```diff
@@ -109,18 +109,21 @@ Display a live stream of one or more containers' resource usage statistics
...
-### unpause
-Unpause one or more containers
-
 ### tag
 Add an additional name to a local image
 
 ### umount
 Unmount a working container's root file system
 
+### unpause
```
Comment (TomSweeneyRedHat) [inline docs/kpod.1.md]: UGH!  ty for fixing.

**Comment by rhatdan** [comment]
Merging.

**Pull request merged**